### PR TITLE
Push translations

### PR DESCRIPTION
**Pull request by mattkime**
Support for uploading translations to transifex.

### CODE REVIEW

File: tests/good-api-test.js [hunk 1]
```diff
@@ -258,6 +258,31 @@ describe("Translations API", function () {
     }).not.throw();
   });
 
+  it("The uploadTranslationInstanceMethod function puts translation content", function (done) {
+    should(function(){
+      nock("https://www.transifex.com")
+        .put("/api/2/project/transifex/resource/sample/translation/es/?file")
+        .reply(201, {
```
Comment (mattkime) [inline tests/good-api-test.js]: @alicoding I spend a bunch of time looking at this and determined that this test passes but there's some side effect that makes further tests fail due to interfering with `nock`

any advice? i'm stuuuuck

**Comment by alicoding** [comment]
Wow sorry I totally missed this! Is this good to merge?

**Comment by mattkime** [comment]
@alicoding if everything looks good to you. we've had it in production for a bit and its been fine there.

**Comment by alicoding** [comment]
Looks good to me and tests are passing now too! 😀

**Pull request merged**